src/screens/subscriber/components/DashboardChartsSection.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+export default class DashboardChartsSection extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeTab: 0
+    };
+    this.switchChartView = this.switchChartView.bind(this);
+  }
+
+  // need to make sure the value doesnt switch if its the same button
+  switchChartView = e => {
+    const { activeTab } = this.state;
+    const { hasShares } = this.props;
+    if (e.target.value != 0 && activeTab === 0 && hasShares) {
```

Comment:
if you want to check that `e.target.value` is not null, just do 
```suggestion
    if (e.target.value && activeTab === 0 && hasShares) {
```

!= should be avoided as much as possible